apps/studio/components/interfaces/Organization/NewOrg/NewOrgForm.tsx

```diff
@@ -58,11 +58,11 @@ const NewOrgForm = ({ onPaymentMethodReset }: NewOrgFormProps) => {
   const stripe = useStripe()
   const elements = useElements()
 
-  const { plan } = useParams()
+  const { plan, name, kind, size, spend_cap } = useParams()
 
-  const [orgName, setOrgName] = useState('')
-  const [orgKind, setOrgKind] = useState(ORG_KIND_DEFAULT)
-  const [orgSize, setOrgSize] = useState(ORG_SIZE_DEFAULT)
+  const [orgName, setOrgName] = useState(name || '')
+  const [orgKind, setOrgKind] = useState(kind || ORG_KIND_DEFAULT)
```

Comment:
low prio since i doubt many will run into this - but we should add validations against `kind` and `size`, if an invalid value is provided in the URL the listbox will not render the option. can probably default to XXX_DEFAULT